### PR TITLE
Changes for transactions logging

### PR DESCRIPTION
**Pull request by mlaflamm**

### CODE REVIEW

File: mry-core/src/main/scala/com/wajam/mry/Database.scala [hunk 1]
```diff
@@ -108,8 +106,7 @@ class Database(var serviceName: String = "database")
 
   private def execute(req: InMessage) {
     var values: Seq[Value] = null
-    val timestamp: Timestamp = req.metadata.getValueOrBuildFromString("timestamp", (v: Seq[String]) => {Timestamp(v(0))}).asInstanceOf[Timestamp]
-    val context = new ExecutionContext(storages, Some(timestamp))
+    val context = new ExecutionContext(storages, Some(Consistency.getMessageTimestamp(req).get))
     context.cluster = Database.this.cluster
```
Comment (coderunner) [inline mry-core/src/main/scala/com/wajam/mry/Database.scala:110]: Shouldn't we think about having timestamp as an attribute in Message one day?

File: mry-core/src/main/scala/com/wajam/mry/Database.scala [hunk 2]
```diff
@@ -117,6 +114,7 @@ class Database(var serviceName: String = "database")
       transaction.execute(context)
       values = context.returnValues
       context.commit()
+      transaction.reset()
```
Comment (coderunner) [inline mry-core/src/main/scala/com/wajam/mry/Database.scala:118]: Why this new line?

File: mry-core/src/main/scala/com/wajam/mry/Database.scala (hunk 1 above)
Comment (mlaflamm) [inline mry-core/src/main/scala/com/wajam/mry/Database.scala:110]: Totally agree once we are serializing Message with protobuf

**Comment by coderunner** [comment]
+1

**Pull request merged**